### ISSUE TITLE
Update PostgreSQL 

### ISSUE THREAD

**Issue by becky-gilbert** [lookit/lookit-api#1445]
# TL;DR

Our PostgreSQL version is very out-of-date, and GCP will stop supporting it in Feb 2025. We might be able to keep using it after that, but we will be switched to "extended support" and charged extra. 

# Summary

PostgreSQL 9.6 was last updated in 2021, and no longer supported by the community. The current version is 16 (see major version history [here](https://www.postgresql.org/support/versioning/)). GCP has emailed to say that they will no longer be supporting versions 9.6 - 12 starting on February 1st 2025. 

> If you do not wish to continue running extended support on your Cloud SQL instances, you can upgrade your Cloud SQL for MySQL instances to MySQL 8.0 or your Cloud SQL for PostgreSQL instances to version 13 or higher before February 1, 2025.

Given that we will need to jump several major releases, we should give ourselves time to do this in case of problems. We should read the upgrading sections in the docs and look at the release notes for all of the major releases since 9.6. (see the "Upgrading" section on the [versioning/releases docs page](https://www.postgresql.org/support/versioning/)). We will need help from Burwood to do the actual update on the cloud versions, so we might want to get in touch with them before the holiday season, in case there are unforeseen problems that we need their help with.
